spec/requests/comments_spec.rb

```diff
@@ -142,6 +149,20 @@
         end
       end
     end
+
+    describe 'DELETE /comment' do
+      it 'deletes anther users comment' do
+        delete idea_comment_path(comment.idea, comment)
+        comment.reload
+        expect(comment.redacted?).to eq true
+      end
+
+      it 'deletes ownd users comment' do
+        delete idea_comment_path(admin_comment.idea, admin_comment)
+        admin_comment.reload
+        expect(admin_comment.redacted?).to eq true
+      end
+    end
   end
```

Comment:
Should it also test that it's not possible to delete another users comment?

Reply:
No because these are tests for admin user so they should be able to edit both there own and another users comment.